packages/components/src/Components/DateFormat/DateFormat.test.js

```diff
@@ -4,6 +4,29 @@ import toJson from 'enzyme-to-json';
 import DateFormat from './DateFormat';
 
 describe('DateFormat component', () => {
+    const _Date = Date;
+    const currDate = new Date('1970');
+    beforeAll(() => {
+        /*eslint no-global-assign:off*/
+        Date = class extends Date {
```

Comment:
I could imagine this mock to be useful for all components/packages. Maybe it could be added in Jest setup instead? This will override the class anyway and if you run jest in parallel it will hit other test files too.